README.md

```diff
@@ -1 +1,28 @@
-# MOHAI_data_app
```

Comment:
Nice looking README. Consider documentation something that you need to learn, so start looking at other APIs and finding those common elements in how documentation comes together to really tell the story about the software and it's usage.